### PR TITLE
Help box reposition

### PR DESCRIPTION
**Pull request by potatowagon**
shift default polition of help box down so it does not block command bar

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -60,9 +61,12 @@ public HelpWindow() {
         helpBox.initModality(Modality.NONE);
         helpBox.setResizable(true);
         helpBox.getDialogPane().setPrefWidth(900.0);
-        helpBox.getDialogPane().setPrefHeight(530.0);
+        helpBox.getDialogPane().setPrefHeight(450.0);
         helpBox.getDialogPane().getStylesheets().add("/view/DarkTheme.css");
         helpBox.getDialogPane().getStyleClass().add("helpBox");
+        helpBox.getDialogPane().setStyle("-fx-font-family: 'Courier New';");
+        helpBox.setX(200);
+        helpBox.setY(125);
```
Comment (pyokagan) [inline src/main/java/seedu/address/ui/HelpWindow.java:69]: @potatowagon ??? Is this relative to the screen? Won't different people's screen make the help window popup at different places?

Comment (potatowagon) [reply src/main/java/seedu/address/ui/HelpWindow.java:69]: im not sure about that, but

screenshot 76

it looks like that on my screen

font is not monospaced :/

Comment (pyokagan) [reply src/main/java/seedu/address/ui/HelpWindow.java:69]: Works for me. Is that on master? I see in your PR you put back `Courier New`
untitled1000

**Comment by pyokagan** [comment]
@potatowagon So what about this PR?

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (potatowagon) [reply src/main/java/seedu/address/ui/HelpWindow.java:69]: yeah, i put it back cause its not working, but after putting it back it still doesnt work, but its okay at least it works on your computer cause all i need is a screenshot to add to the UG. And yes its from current master

**Pull request closed without merging**